src/forms/checkbox.test.jsx

```diff
@@ -20,52 +21,106 @@ describe('Checkbox', function() {
 		checkbox = null;
 	});
 
-	it('exists', function() {
-		expect(() => TestUtils.findRenderedDOMComponentWithTag(checkboxComponent, 'input')).not.toThrow();
+	describe('basic', function() {
+		it('exists', function() {
+			expect(() => TestUtils.findRenderedDOMComponentWithTag(checkboxComponent, 'input')).not.toThrow();
```

Comment:
we should change these tests to use `enzyme`. The `exists` test can use snapshotting. See https://github.com/meetup/meetup-web-components/pull/248

also you can do a `shallow` render of the checkbox component and then look for attributes and other behavior. We don't need to test all props/attributes of the element if we are confident the snapshot rendered correctly.
 https://github.com/meetup/meetup-web-components/pull/248/files#diff-2690983ef9c689ac0361949ee7bfd4d9R41